rating-display.js

```diff
@@ -0,0 +1,299 @@
+import {
+  LitElement,
+  html,
+  css,
+} from "https://unpkg.com/lit-element/lit-element.js?module";
+
+/* ----------------------------------------------------------------- */
+const googleApiKey = "AIzaSyAJ9pMGaHcmOiNeHEXQLGCiJcr5k3TV4F8"; // Google API Key
```

Comment:
If these variables are moved to the config file, why are they still declared here?

Reply:
It was due to the fact that, while building the experiment, the local file wasn't referenced.